docs/persistent_graph_store.rst

```diff
@@ -0,0 +1,516 @@
+| skip.header           | false    | true    | Whether to skip the first line of the input file                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
++-----------------------+----------+---------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+
+After data building completed, you can find the output files in the `output.path` of HDFS. The output files includes a meta file named `META`, an empty file named `_SUCCESS`, and some data files that one for each partition named in the pattern of `part-r-xxxxx.sst`. The layout of the output directory should look like:
+
+.. code:: plain-text
+    /tmp/data_output
+      |- META
+      |- _SUCCESS
+      |- part-r-00000.sst
+      |- part-r-00001.sst
+      |- part-r-00002.sst
+      ...
+
+
+2. Loading graph partitions
+""""""""""""""""""""
+
+Now ingest the offline built data into the graph storage:
+
+NOTE: You need to make sure that the HDFS endpoint that can be accessed from the processes of the graph store.
+
+.. code:: bash
+
+    $ ./load_tool.sh -c ingest -d hdfs://1.2.3.4:9000/tmp/data_output
+
+
+The offline built data can be ingested successfully only once, otherwise errors will occur.
+
+
+3. Commit to store service
+""""""""""""""""""""""""""
+
+After data ingested into graph storage, you need to commit data loading. The data will not be able to read until committed successfully.
+
+.. code:: bash
+
+    $ ./load_tool.sh -c commit -d hdfs://1.2.3.4:9000/tmp/data_output
+
+Notice: The later committed data will overwrite the earlier committed data which have same vertex types or edge relations.
+
+
+Running Gremlin Queries
```

Comment:
I think this should be placed before realtime write, as the update or delete edges involves the queries.

Reply:
done